web-ui/src/main/resources/catalog/components/edit/associatedmd/AssociatedMdDirective.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
would have called it `EntryAssociateMd` cause we already have a lot of stuffs for associated resources/metadata that is a bit different.